docs/examples/merging/record.json

```diff
@@ -1,60 +1,268 @@
 {
-  "uri": "https://standard.open-contracting.org/examples/records/ocds-213czf-000-00002-merge.json",
+  "uri": "https://standard.open-contracting.org/examples/rcords/ocds-213czf-000-00002-01.json",
```

Comment:
Why did we rename this file? I prefer to leave it as it was, to avoid changing a file under `schema/`.